app/utils.ts

```diff
@@ -172,154 +170,197 @@ export function getRpcUrl(network: string) {
-        imageUrl: "/logos/usdt-logo.svg",
-      },
-      {
-        name: "USD Coin",
-        symbol: "USDC",
-        decimals: 18,
-        address: "0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d",
-        imageUrl: "/logos/usdc-logo.svg",
-      },
-      {
-        name: "cNGN",
-        symbol: "cNGN",
-        decimals: 6,
-        address: "0xa8aea66b361a8d53e8865c62d142167af28af058",
-        imageUrl: "/logos/cngn-logo.svg",
-      },
-    ],
-    Polygon: [
-      {
-        name: "USD Coin",
-        symbol: "USDC",
-        decimals: 6,
-        address: "0x3c499c542cef5e3811e1192ce70d8cc03d5c3359",
-        imageUrl: "/logos/usdc-logo.svg",
-      },
-      {
-        name: "Tether USD",
-        symbol: "USDT",
-        decimals: 6,
-        address: "0xc2132d05d31c914a87c6611c10748aeb04b58e8f",
-        imageUrl: "/logos/usdt-logo.svg",
-      },
-      {
-        name: "cNGN",
-        symbol: "cNGN",
-        decimals: 6,
-        address: "0x52828daa48c1a9a06f37500882b42daf0be04c3b",
-        imageUrl: "/logos/cngn-logo.svg",
-      },
-    ],
-    Scroll: [
-      {
-        name: "USD Coin",
-        symbol: "USDC",
-        decimals: 6,
-        address: "0x06eFdBFf2a14a7c8E15944D1F4A48F9F95F663A4",
-        imageUrl: "/logos/usdc-logo.svg",
-      },
-    ],
-    Optimism: [
-      {
-        name: "USD Coin",
-        symbol: "USDC",
-        decimals: 6,
-        address: "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
-        imageUrl: "/logos/usdc-logo.svg",
-      },
-    ],
-    Celo: [
-      {
-        name: "Tether USD",
-        symbol: "USDT",
-        decimals: 6,
-        address: "0x48065fbBE25f71C9282ddf5e1cD6D6A887483D5e",
-        imageUrl: "/logos/usdt-logo.svg",
-      },
-      {
-        name: "USD Coin",
-        symbol: "USDC",
-        decimals: 6,
-        address: "0xcebA9300f2b948710d2653dD7B07f33A8B32118C",
-        imageUrl: "/logos/usdc-logo.svg",
-      },
-      {
-        name: "Celo Dollar",
-        symbol: "cUSD",
-        decimals: 18,
-        address: "0x765DE816845861e75A25fCA122bb6898B8B1282a",
-        imageUrl: "/logos/cusd-logo.svg",
-      },
-    ],
-    Lisk: [
-      {
-        name: "Tether USD",
-        symbol: "USDT",
-        decimals: 6,
-        address: "0x05D032ac25d322df992303dCa074EE7392C117b9",
-        imageUrl: "/logos/usdt-logo.svg",
-      },
-    ],
+ * Converts API network identifiers to display names
+ * @param networkId - Network identifier from API (e.g., "arbitrum-one")
+ * @returns Display name (e.g., "Arbitrum One")
+ */
+export function normalizeNetworkName(networkId: string): string {
+  const networkNames: Record<string, string> = {
+    "arbitrum-one": "Arbitrum One",
+    base: "Base",
+    polygon: "Polygon",
+    "bnb-smart-chain": "BNB Smart Chain",
+    celo: "Celo",
+    lisk: "Lisk",
+    scroll: "Scroll",
+    optimism: "Optimism",
+  };
+
+  return networkNames[networkId] || networkId;
+}
```

Comment:
@coderabbitai make the `normalizeNetworkName` function dynamic and not limited to a fixed network definition